examples/lujain_work_area/Contact_Book/main.py

```diff
@@ -0,0 +1,103 @@
+# main.py
+
+import csv
+import re
+from contact_book import ContactBook
+
+def read_contact_records(csv_file):
+    with open(csv_file, mode='r') as file:
+        reader = csv.DictReader(file)
+        for row in reader:
+            print(row)
+
+def validate_phone(phone):
+    # Regular expression for validating phone numbers with optional area code and separator
+    phone_pattern = r'^(\+\d{1,3})?[-\s]?\d{3,}[-\s]?\d{3,}$'
+    return re.match(phone_pattern, phone) is not None
+
+def validate_email(email):
+    # Regular expression for validating email addresses
+    email_pattern = r'^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$'
+    return re.match(email_pattern, email)
+
+def validate_name(name):
```

Comment:
ContactRecord should be responsible for validating the data while the object is being created to assigned any data